src/Plutarch/Extra/Precompile.hs

```diff
@@ -0,0 +1,218 @@
+{-# LANGUAGE OverloadedRecordDot #-}
+{-# LANGUAGE RankNTypes #-}
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+
+{-# HLINT ignore "Use camelCase" #-}
```

Comment:
Explanatory note would be good.

Reply:
This actually can be removed, I think. I think it had to do with the `LiftError`, which had underscores (because it was mimicking the upstream)